RedHatInsights/insights-remediations-frontend
# ESSNTL-1136: Update links for execute modal

This PR fixes the informational links in the Execute Modal. Before, these links were unresponsive, now they properly link to the correct places as outlined in the Jira card.

To test this:

- Create a remediation playbook
  - You can easily do this by going to the remediations page and if none are available you will have links to other apps to create one.
  - Signing into insights account in stage will also provide you with existing remediation playbooks.
- Click a remediation playbook
- Click the button "Execute playbook"

You should see "Learn more about Cloud Connector"; and in the dropdown "About remote execution with Cloud connector" you will find two more links, "How to configure Receptor/Cloud Connector on Red Hat Satellite" and "How to enable Cloud Connector with Red Hat connect".

These links were previous unresponsive, but work with this PR.
